### ISSUE TITLE
Can bootforce be used In VisualForce

### ISSUE THREAD

**Issue by BourhiMaryem** [swervo/bootforce#4]
i Have tried to include Bootforce.js in a visualforce page, but i keep getting errors, which led me to ask the question can this library be used within a visualforce page ?

**Comment by swervo** [comment]
Yes. Should be possible. It is only a javascript file after all. What errors are you seeing?
